weblate/templates/zen-units.html

```diff
@@ -88,7 +88,10 @@
 </td>
 {% endif %}
 <td {% if user.profile.zen_mode != user.profile.ZEN_HORIZONTAL %}colspan="2"{% endif %} class="translator">
-{% crispy item.form %}
+  <button id="hide-sticky" class="btn btn-light btn-sm" title="{% trans "Collapse" %}" >
```

Comment:
This form is there for each string, so using `id` is probably not a good idea as there will be more of them in the page.

Reply:
Yeah that's not a problem in this case, we select only the closest hide btn to the string editor we are in. So others are ignored

Reply:
Isn't `id` supposed to be unique in the document?

Reply:
Yes that's what is said. But to be honest I'm not sure if it is that important or enforced by any means.